### PR TITLE
fix: margin right on warning icon to 8px

### PR DESCRIPTION
**Pull request by stellalc7**
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen_Shot_2021-06-05_at_12 31 12_PM_(1)
Screenshot 2021-07-15 at 14 40 15

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by AAfghahi — APPROVED** [review]

File: superset-frontend/src/components/WarningIconWithTooltip/index.tsx [hunk 1]
```diff
@@ -35,7 +35,10 @@ function WarningIconWithTooltip({
       id="warning-tooltip"
       title={<SafeMarkdown source={warningMarkdown} />}
     >
-      <Icons.AlertSolid iconColor={theme.colors.alert.base} />
+      <Icons.AlertSolid
+        iconColor={theme.colors.alert.base}
+        css={{ marginRight: `8px` }}
```
Comment (suddjian) [inline superset-frontend/src/components/WarningIconWithTooltip/index.tsx:40]: Let's use grid units.
```suggestion
        css={{ marginRight: `${theme.gridUnit * 2}px` }}
```

**Review by suddjian — CHANGES_REQUESTED** [review]
Thanks for the fix! Happy to merge once pixel values are in grid units.

File: superset-frontend/src/components/WarningIconWithTooltip/index.tsx (hunk 1 above)
Comment (stellalc7) [reply superset-frontend/src/components/WarningIconWithTooltip/index.tsx]: Oh, I just saw your exact suggestion - let me know if what I coded is not exactly correct. Thanks!

**Review by geido — APPROVED** [review]
LGTM! Thanks for the contribution

**Comment by stellalc7** [comment]
hey @suddjian! did you need me to do anything else on this PR? :)

**Review by suddjian — APPROVED** [review]
Looks good, thank you!

**Pull request merged**